src/uwtools/templater.py

```diff
@@ -104,16 +108,15 @@ def set_template(argv):
         log.handlers.clear()
         log.propagate = False
 
-
-
-    log.info(f"""Running script templater.py with args:
-{('-' * 70)}
-{('-' * 70)}""")
-    for name, val in user_args.__dict__.items():
-        if name not in ["config"]:
-            log.info("{name:>15s}: {val}".format(name=name, val=val))
-    log.info(f"""{('-' * 70)}
-{('-' * 70)}""")
+    log_details(__name__, user_args)
+#    log.info(f"""Running script templater.py with args:
+#{('-' * 70)}
+#{('-' * 70)}""")
+#    for name, val in user_args.__dict__.items():
+#        if name not in ["config"]:
+#            log.info("{name:>15s}: {val}".format(name=name, val=val))
+#    log.info(f"""{('-' * 70)}
+#{('-' * 70)}""")
 
 
     if user_args.config_file:
```

Comment:
Instead of adding a new function to the logger here, I'd like to see something like passing the templater logger name to the YAMLConfig class here, and then maybe in the YAMLConfig __init__ , you could get the logger, and report that it's creating a config object and list the args that were passed.

Reply:
This is really why I kept this on the draft - I felt that the log block here from templater.py covered what we needed, but I was wary of changing that part and making a new function. I think to pass the name variable, the right way would be with a decorator, otherwise I think I'd have to add another arg to all of the configs

Reply:
Just the base config....